testing/test_mark.py

```diff
@@ -1107,6 +1107,25 @@ def test_parameterset_for_parametrize_marks(testdir, mark):
         assert result_mark.kwargs.get("run") is False
 
 
+def test_parameterset_for_fail_at_parametrize(testdir):
+    testdir.makeini(
+        "[pytest]\n{}={}".format(EMPTY_PARAMETERSET_OPTION, "fail_at_parametrize")
```

Comment:
This can be written as:

```python
testdir.makeini("""
    [pytest]
    {}=fail_at_parametrize
""".format(EMPTY_PARAMETERSET_OPTION))
```

Which is a more readable IMHO. 👍

Reply:
Agreed, but that's a copy paste from the test above, so I change both.